### PR TITLE
AB#65910 moved common Workflow logic from Loans + Type of housing AHP view 

### PR DESCRIPTION
**Pull request by jjankowski87**
image

### CODE REVIEW

File: src/HE.Investment.AHP.WWW/Views/HomeTypes/Index.cshtml.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+using Microsoft.AspNetCore.Mvc.RazorPages;
```
Comment (Antek012) [inline src/HE.Investment.AHP.WWW/Views/HomeTypes/Index.cshtml.cs:1]: Is this file needed?

File: src/HE.Investment.AHP.WWW/Views/HomeTypes/TypeOfHousing.cshtml.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+using Microsoft.AspNetCore.Mvc.RazorPages;
```
Comment (Antek012) [inline src/HE.Investment.AHP.WWW/Views/HomeTypes/TypeOfHousing.cshtml.cs:1]: also here?

**Review by Antek012 — APPROVED** [review]

**Review by Carq — APPROVED** [review]

File: src/HE.Investment.AHP.WWW/Views/Shared/_ErrorSummaryPartial.cshtml
[the remark this answers is not in the document]
Comment (Antek012) [reply src/HE.Investment.AHP.WWW/Views/Shared/_ErrorSummaryPartial.cshtml:1]: @jjankowski87 in my latest PR you can find solution how to access view from shared project.

File: src/HE.Investment.AHP.Domain/HomeTypes/HomeTypesWorkflow.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
+using HE.InvestmentLoans.BusinessLogic.LoanApplication;
+using HE.InvestmentLoans.Common.Routing;
+using Stateless;
+
+namespace HE.Investment.AHP.BusinessLogic.HomeTypes;
+
+public class HomeTypesWorkflow : IStateRouting<HomeTypesWorkflow.State>
+{
+    public enum State
```
[the remark this answers is not in the document]
Comment (jjankowski87) [reply src/HE.Investment.AHP.Domain/HomeTypes/HomeTypesWorkflow.cs:9]: will do

File: src/HE.Investment.AHP.WWW/Controllers/HomeTypesController.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+using HE.InvestmentLoans.Common.Routing;
+using HE.Investments.Common.WWW.Routing;
+using Microsoft.AspNetCore.Mvc;
+
+namespace HE.Investment.AHP.WWW.Controllers;
+
+[Route("home-types")]
+public class HomeTypesController : WorkflowController<HomeTypesWorkflow.State>
+{
+    [HttpGet("index")]
```
[the remark this answers is not in the document]
Comment (jjankowski87) [reply src/HE.Investment.AHP.WWW/Controllers/HomeTypesController.cs:12]: sure

File: src/HE.Investment.AHP.WWW/Models/HomeTypes/TypeOfHousingModel.cs [hunk 5]
```diff
@@ -0,0 +1,15 @@
+namespace HE.Investment.AHP.WWW.Models.HomeTypes;
+
+public class TypeOfHousingModel
```
[the remark this answers is not in the document]
Comment (jjankowski87) [reply src/HE.Investment.AHP.WWW/Models/HomeTypes/TypeOfHousingModel.cs:3]: ok

File: src/HE.Investment.AHP.WWW/Views/HomeTypes/TypeOfHousing.cshtml.cs [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+
+namespace HE.Investment.AHP.WWW.Views.HomeTypes;
+
+public class TypeOfHousing : PageModel
```
[the remark this answers is not in the document]
Comment (jjankowski87) [reply src/HE.Investment.AHP.WWW/Views/HomeTypes/TypeOfHousing.cshtml.cs:5]: ok, will be removed

File: src/HE.Investment.AHP.WWW/Views/Shared/_ErrorSummaryPartial.cshtml
[the remark this answers is not in the document]
Comment (jjankowski87) [reply src/HE.Investment.AHP.WWW/Views/Shared/_ErrorSummaryPartial.cshtml:1]: ok, I'll do this in next PR

File: src/HE.Investment.AHP.WWW/Views/HomeTypes/Index.cshtml.cs (hunk 1 above)
Comment (jjankowski87) [reply src/HE.Investment.AHP.WWW/Views/HomeTypes/Index.cshtml.cs:1]: removed this file as Carq suggested

File: src/HE.Investment.AHP.WWW/Views/HomeTypes/TypeOfHousing.cshtml.cs (hunk 2 above)
Comment (jjankowski87) [reply src/HE.Investment.AHP.WWW/Views/HomeTypes/TypeOfHousing.cshtml.cs:1]: removed this file as Carq suggested
